### PR TITLE
Update install-kubectl.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with English version.

### CODE REVIEW

File: content/zh/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -234,15 +234,24 @@ If you are on macOS and using [Homebrew](https://brew.sh/) package manager, you
...
 1. 运行安装命令：
 
+   ```bash
+   brew install kubectl 
    ```
+   
+<!--
+   or
+-->
+   或者
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl.md:252]: 请将249、252 行缩进三个空格。避免列表解析错误。
根据目前的预览看，这里不缩进的话，整个结构是错的。
https://deploy-preview-25662--kubernetes-io-master-staging.netlify.app/zh/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos

Comment (guzj11) [reply content/zh/docs/tasks/tools/install-kubectl.md:252]: 已按照建议修改

File: content/zh/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -297,7 +307,7 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
    Or if you have `curl` installed, use this command:
 -->
 1. 从[此链接](https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe)
-   下载最新发型版本。
+   下载最新发型版本 {{< param "fullversion" >}}。
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-kubectl.md:310]: 发行

Comment (guzj11) [reply content/zh/docs/tasks/tools/install-kubectl.md]: suggestion applied
commits have been suqashed

**Comment by guzj11** [comment]
> 
> 
> /approve

can we merge this commit?

**Comment by tengqm** [comment]
@guzj11 Please be patient. It would be good to have other reviewers to review and LGTM this.

**Review by tanjunchen — COMMENTED** [review]
thanks
/lgtm

**Pull request merged**